src/components/base/Notification/Notification.module.scss

```diff
@@ -28,6 +29,14 @@
     --padding-block-large: 48;
   }
 
+  @include forced-colors {
+    border: px-to-rem(var(--border-width-m)) solid CanvasText;
+  }
+
+  &.borderless {
+    border: none;
```

Comment:
We need to add the border in the forced-colors mode for the borderless variant as well, even if it sounds a bit contradictory:-)

```
@include forced-colors {
    border: px-to-rem(var(--border-width-m)) solid CanvasText;
  }
```